src/osm_callbacks.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+#ifndef TIMESCALEDB_OSM_CALLBACKS_H
+#define TIMESCALEDB_OSM_CALLBACKS_H
+
+#include <postgres.h>
+#include <catalog/objectaddress.h>
+
+typedef int (*chunk_insert_check_hook_type)(Oid ht_oid, int64 range_start, int64 range_end);
+typedef void (*hypertable_drop_hook_type)(const char *schema_name, const char *table_name);
+
+typedef struct
+{
+	chunk_insert_check_hook_type chunk_insert_check_hook;
```

Comment:
please document what the hook is for - looking at the if in the other file ; I'm a bit confused what its contract could be